include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -1738,6 +1738,14 @@ struct ConstValueImpl : Base<T> {
   /// <returns>byte length for the specified string element</returns>
   size_t GetStringTensorElementLength(size_t element_index) const;
 
+  /// <summary>
+  /// This API returns the size of the tensor in bytes iff this ort value
+  /// contains a tensor or primitive type. The API would throw an exception if the OrtValue
```

Comment:
I don't quite understand "a tensor or primitive type". doesn't "tensor size" only make sense for tensors?